src/cpp/server/server_cc.cc

```diff
@@ -338,28 +338,27 @@ class Server::SyncRequestThreadManager : public ThreadManager {
   }
 
   void Shutdown() override {
-    ThreadManager::Shutdown();
     server_cq_->Shutdown();
+    ThreadManager::Shutdown();
```

Comment:
Why is this reordered?

Reply:
Stylistic reasons; I wanted to see the derived class's shutdown actions being done before the base class's. This is purely stylistic here as these just initiate shutdown actions and there does not need to be any ordering.